### ISSUE TITLE
Best in game connectivity

### ISSUE THREAD

**Issue by Basker-Ville** [0x1c1101/itamiproxy#3]
I am not a coding expert but, this proxy has a really nice connectivity, it uses a unique way of connection which is different than the usual which is by a client cli? I've been trying to add some features, I already have a proxy, that I modified a bit but the connectivity is very bad, but nothing has been working, since this proxy is pure. When I create events.cpp, and try to add the commands, and end up by adding up a lot of things, no errors are shown visually, but when i build it i get like 500 errors. I am not a coder, I do this for fun, but if you could help me, just adding one command in events.cpp, so that i could continue off by myself it would be great. Thanks.
(Note: I want to use it for personnel fun, not to sell, nor to steal with it)

**Comment by 0x1c1101** [comment]
There already are /help and /warp <world_name> commands, You can take these as example.

**Comment by Basker-Ville** [comment]
> There already are /help and /warp <world_name> commands, You can take these as example.

Yes I've seen these, but it seems like this proxy has commands in a untraditional way(nothing wrong here) and I tried adding a command which was antigravity, I added everything that i needed, world.h, player.h and all of these but i kept gettibg endless errors. So is there any other way?

**Comment by 0x1c1101** [comment]
What are the errors?

**Comment by Basker-Ville** [comment]
The errors start when i start adding world.h and player.h which qre still a little basic (these 2 work fine in my other proxy) but here when i add them i start getting errors i fix to the way of adaption of the proxy and then no visual errors are shown when i build it i get like 300 errors, but when i click on one it directs me to world.h or player.h, but then nothing is shown

**Comment by Basker-Ville** [comment]
Yes, i know that i need to change things, I did include the files and everything, and that i can't just copy, and paste, i did try to make it compatible with the proxy, but there's a lot of errors of many types still. I won't try to fix it anymore, anyways good proxy, keep it up.
